### PR TITLE
v0.8.0 blog post

### PR DESCRIPTION
**Pull request by Mytherin**
Supersedes #752

Please have a look

### CODE REVIEW

**Comment by Alex-Monahan** [comment]
Hi @Mytherin! Thanks so much for putting this together! I made some changes, but feel free to undo any that you are not excited about. If you have any details about the data size for each micro-benchmark, I think it would be great to include that too.

**Comment by hannes** [comment]
Man I had the repro in a fold-out thing even :D

**Comment by Mytherin** [comment]
Maybe we can add an `Appendix: Experiment Info` or so? I would prefer not to have it inline since it distracts from the release notes - and I think most people don't care esp as we are only comparing against ourselves

**Comment by hannes** [comment]
ah lets just leave it as is its fine

File: _posts/2023-05-17-announcing-duckdb-080.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+| 2023-05-15 11:31:00 | 2023-05-15 11:30:00 |
+
+
+Please [refer to the documentation](/docs/guides/sql_features/asof_join.html) for a more in-depth explanation.
+
+
+
+#### Data Integration Improvements
+
+[**Default Parallel CSV Reader.**](https://github.com/duckdb/duckdb/pull/6977). In this release, the parallel CSV reader has been vastly improved and is now the default CSV reader. We would like to thank everyone that has tried out the experimental reader for their valuable feedback and reports. The `experimental_parallel_csv` flag has been deprecated and is no longer required. The parallel CSV reader enables much more efficient reading of large CSV files.
```
Comment (carlopi) [inline _posts/2023-05-17-announcing-duckdb-080.md:107]:
```suggestion
[**Default Parallel CSV Reader**](https://github.com/duckdb/duckdb/pull/6977). In this release, the parallel CSV reader has been vastly improved and is now the default CSV reader. We would like to thank everyone that has tried out the experimental reader for their valuable feedback and reports. The `experimental_parallel_csv` flag has been deprecated and is no longer required. The parallel CSV reader enables much more efficient reading of large CSV files. 
```

File: _posts/2023-05-17-announcing-duckdb-080.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+```sql
+FROM 'data/glob/crawl/stackoverflow/**/*.csv';
+```
+
+[The documentation has been updated](/docs/data/multiple_files/overview) with various examples of this syntax.
+
+
+#### Storage Improvements
+
+[**Lazy-Loading Table Metadata**.](https://github.com/duckdb/duckdb/pull/6715). DuckDB’s internal storage format stores metadata for every row group in a table, such as min-max indices and where in the file every row group is stored. In the past, DuckDB would load this metadata immediately once the database was opened. However, once the data gets very big, the metadata can also get quite large, leading to a noticeable delay on database startup. In this release, we have optimized the metadata handling of DuckDB to only read table metadata as its being accessed. As a result, startup is near-instantaneous even for large databases, and metadata is only loaded for columns that are actually used in queries. The benchmarks below are for a database file containing a single large TPC-H `lineitem` table (120x SF1) with ~770 million rows and 16 columns:
```
Comment (carlopi) [inline _posts/2023-05-17-announcing-duckdb-080.md:144]:
```suggestion
[**Lazy-Loading Table Metadata**](https://github.com/duckdb/duckdb/pull/6715). DuckDB’s internal storage format stores metadata for every row group in a table, such as min-max indices and where in the file every row group is stored. In the past, DuckDB would load this metadata immediately once the database was opened. However, once the data gets very big, the metadata can also get quite large, leading to a noticeable delay on database startup. In this release, we have optimized the metadata handling of DuckDB to only read table metadata as its being accessed. As a result, startup is near-instantaneous even for large databases, and metadata is only loaded for columns that are actually used in queries. The benchmarks below are for a database file containing a single large TPC-H `lineitem` table (120x SF1) with ~770 million rows and 16 columns:
```

File: _posts/2023-05-17-announcing-duckdb-080.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+```sql
+CREATE TABLE lineitem AS FROM lineitem.csv;
+```
+
+| v0.7.1 | v0.8.0 |
+|--------|--------|
+| 4.1s   | 1.2s   |
+
+**Parallel [Parquet](https://github.com/duckdb/duckdb/pull/7375), [CSV and JSON Writing](https://github.com/duckdb/duckdb/pull/7368).** This release includes support for parallel *order-preserving* writing of Parquet, CSV and JSON files. As a result, writing to these file formats is parallel by default, also without disabling insertion order preservation, and writing to these formats is greatly sped up.
```
Comment (carlopi) [inline _posts/2023-05-17-announcing-duckdb-080.md:119]:
```suggestion
**Parallel [Parquet](https://github.com/duckdb/duckdb/pull/7375), [CSV and JSON Writing](https://github.com/duckdb/duckdb/pull/7368)**. This release includes support for parallel *order-preserving* writing of Parquet, CSV and JSON files. As a result, writing to these file formats is parallel by default, also without disabling insertion order preservation, and writing to these formats is greatly sped up.
```

**Review by carlopi — COMMENTED** [review]
3 trivial formatting issue.

I would also consider adding a brief paragraph just before the end about extensions, spatial in particular seems worth sharing.

File: _posts/2023-05-17-announcing-duckdb-080.md (hunk 1 above)
Comment (hannes) [reply _posts/2023-05-17-announcing-duckdb-080.md:107]: Spatial had its own blog post, which is why I left it out here

**Pull request merged**